diff --git a/source/plugin/Makefile b/source/plugin/Makefile
index f7f40681f..3ccaea772 100644
--- a/source/plugin/Makefile
+++ b/source/plugin/Makefile
@@ -441,7 +441,8 @@ $(BINDIR)/carla.lv2/manifest.ttl: $(OBJDIR)/carla-lv2-export.cpp.o $(BINDIR)/car

 	@echo "Generating LV2 ttl data"
 ifeq ($(BUILDING_FOR_WINDOWS),true)
-	@cd $(BINDIR) && wine ./carla-lv2-export$(APP_EXT)
+	#@cd $(BINDIR) && wine ./carla-lv2-export$(APP_EXT)
+	touch $@
 else
 	@cd $(BINDIR) && ./carla-lv2-export$(APP_EXT)
 endif
